.php_cs

```diff
@@ -86,38 +87,37 @@ $config->setRules(
         'no_unneeded_final_method' => true,
         'no_unused_imports' => true,
         'no_useless_else' => true,
-        'no_useless_return' => true,
         'no_whitespace_before_comma_in_array' => true,
         'no_whitespace_in_blank_line' => true,
-        'non_printable_character' => ['use_escape_sequences_in_strings' => true],
         'normalize_index_brace' => true,
+        'object_operator_without_whitespace' => true,
         'ordered_imports' => true,
         'phpdoc_add_missing_param_annotation' => true,
         'phpdoc_indent' => true,
         'phpdoc_no_access' => true,
-        'phpdoc_no_empty_return' => true,
```

Comment:
👎

Reply:
There should always be `@return` in docblock and whether it's null or a void is a valuable information.

Reply:
I disagree.  IF something isn't returning something, it shouldn't have a return annotation.

Reply:
Maybe I'm just used to having those. PHPDoc will assume `@return void` if it's not present and I'm happy with that.